megatron/arguments.py

```diff
@@ -365,6 +370,13 @@ def _add_learning_rate_args(parser):
                             'rate, maximum number of iterations, and decay style '
                             'from checkpoint and ignore input arguments.')
 
+    # Adafactor
+    group.add_argument('--adafactor-decay', type=float, default=-0.8, help='Coefficient used to compute running averages of square')
+    group.add_argument('--relative-step', action='store_true', help='If True, time-dependent learning rate is computed instead of external learning rate')
+    group.add_argument('--scale-parameter', action='store_true', help='If True, learning rate is scaled by root mean square')
+    group.add_argument('--adafactor-warmup', action='store_true', help='Time-dependent learning rate computation depends on whether warm-up initialization is being used')
```

Comment:
I would structure the help-string like the others: `If True, ...."

Reply:
@StellaAthena what do you think about completely delete one of the parameters `relative_step` and `warmup_init`? I can change the code of Adafactor accordingly, in order to use just one parameter (as you noticed in the Discord channel, one of them is useless), and then add the modified optimizer in `optimizers.py`? I think `relative_step` only would be ok.